### ISSUE TITLE
RelayDistribution

### ISSUE THREAD

**Issue by NielsZeilemaker** [Tribler/dispersy#164]
I've found the RelayDistribution after getting an assert, however it doens't seem to be implemented at all?
Any plans on implementing it, or should it be removed?

**Comment by boudewijn-tribler** [comment]
Correct, it is not implemented.  As far as I know there is no immediate plan to implement this, although we should check with Vincent, he was looking at this feature a few weeks ago.

**Comment by NielsZeilemaker** [comment]
@whirm let's remove this.
